bitnami/cassandra/4.1/debian-11/rootfs/opt/bitnami/scripts/libcassandra.sh

```diff
@@ -233,6 +233,10 @@ cassandra_validate() {
         check_resolved_hostname "$seed"
     done
 
+    if [[ ! ${CASSANDRA_AUTO_SNAPSHOT_TTL} =~ ([1-9]{1}[0-9]{0,}[d|h|m]) ]]; then
+       print_validation_error "CASSANDRA_AUTO_SNAPSHOT_TTL accepted units: d (days), h (hours) or m (minutes)"
```

Comment:
Could you fix the indentation?

```suggestion
        print_validation_error "CASSANDRA_AUTO_SNAPSHOT_TTL accepted units: d (days), h (hours) or m (minutes)"
```